stencil-library/storybook-pages/angular.stories.mdx

```diff
@@ -0,0 +1,189 @@
+        src='https://cdn.jsdelivr.net/npm/@justifi/webcomponents@4.6.0/dist/webcomponents/webcomponents.esm.js'></script>
+    </head>
+  `)}
+/>
+
+2. Update Angular Module Configuration:
+
+<Source
+  language="typescript"
+  dark
+  code={dedent(`
+    import { NgModule, CUSTOM_ELEMENTS_SCHEMA } from '@angular/core';
+    import { BrowserModule } from '@angular/platform-browser';
+    import { AppComponent } from './app.component';
+
+    @NgModule({
+      declarations: [
+        AppComponent
+      ],
+      imports: [
+        BrowserModule
+      ],
+      providers: [],
+      bootstrap: [AppComponent],
+      schemas: [
+        CUSTOM_ELEMENTS_SCHEMA
+      ]
+    })
+    export class AppModule { }
+
+`)}
+/>
+
+## Props and Event Handling
+
+- **Props**: Use Angular template syntax for passing props to components. For example, `validationMode` accepts `onBlur`, `onChange` or `onSubmit`. It's also possible to pass variables:
+
+<Source
+  language="html"
+  dark
+  code={dedent(`
+    <justifi-card-form
+      [validationMode]="'onBlur'"
+    >
+  `)}
+/>
+
+- **Event Handling**: Listen to events using Angular's event binding syntax.
+
+<Source
+  language="html"
+  dark
+  code={dedent(`
+    <justifi-card-form
+      (cardFormReady)="onCardFormReady($event)"
+      (cardFormTokenize)="onTokenize($event)"
+    >
+  `)}
+/>
+
+## Calling methods
+
+To programmatically call methods on web components from an Angular component, you should first obtain a reference to the web component instance. This can be achieved using Angular’s ViewChild decorator.
+
+1. **Reference the element**: Use `ViewChild` to get a reference to the web component in your Angular component. Assign a template reference variable (`#refVariable`) to the web component in your template, and then access it in your TypeScript code.
+
+In your template (`app.component.html`):
+
+<Source
+  language="html"
+  dark
+  code={dedent(`
+    <justifi-card-form
+      #cardForm
+      [validationMode]="'onBlur'"
+      (cardFormReady)="onCardFormReady($event)"
+      (ready)="onReady($event)"
+      (cardFormTokenize)="onTokenize($event)"
+    >
+    <button (click)="handleValidate()">Validate</button>
+`)}
+/>
+
+In your component (`app.component.ts`):
+
+<Source
+  language="typescript"
+  dark
+  code={dedent(`
+    import { Component, ViewChild, ElementRef, AfterViewInit } from '@angular/core';
+
+    @Component({
+      selector: 'app-root',
+      templateUrl: './app.component.html',
+      styleUrls: ['./app.component.css']
+    })
+    export class AppComponent implements AfterViewInit {
+      @ViewChild('cardForm') cardForm!: ElementRef;
+
+      handleValidate() {
+        // get a ref to the component and call validate method
+        this.cardForm.nativeElement.validate();
+
+        console.log('onValidate', event);
+      }
+    }
+
+`)}
+/>
+
+Remember that the web component's methods will be available on the nativeElement of the ElementRef.
```

Comment:
This and the following line are some really nice details to have here, nice work 👍